apollo-adminservice/src/test/java/com/ctrip/framework/apollo/adminservice/controller/ItemSetControllerTest.java

```diff
@@ -95,16 +91,18 @@ public void testItemSetCreatedWithInvalidNamespaceId() {
     Assert.assertNotNull(someNamespace);
     long someNamespaceId = someNamespace.getId();
 
-    RestTemplate createdTemplate = (new TestRestTemplate()).getRestTemplate();
-    createdTemplate.setMessageConverters(restTemplate.getMessageConverters());
-
     int createdSize = 3;
     ItemChangeSets itemSet = mockCreateItemChangeSets(namespace, createdSize);
     itemSet.getCreateItems().get(createdSize - 1).setNamespaceId(someNamespaceId);
 
-    ResponseEntity<Void> response =
-        createdTemplate.postForEntity(itemSetBaseUrl(), itemSet, Void.class, appId, clusterName, namespaceName);
-    Assert.assertEquals(HttpStatus.BAD_REQUEST, response.getStatusCode());
+    try {
+      ResponseEntity<Void> response =
+          restTemplate.postForEntity(itemSetBaseUrl(), itemSet, Void.class, appId, clusterName, namespaceName);
+      Assert.assertEquals(HttpStatus.BAD_REQUEST, response.getStatusCode());
```

Comment:
This could be put in the catch block? e.g. 

```java
    catch (HttpClientErrorException e){
      Assert.assertEquals(HttpStatus.BAD_REQUEST, e.getStatusCode());
    }
```